### PR TITLE
Update playerlist.js

### PR DESCRIPTION
**Pull request by u9g**

### CODE REVIEW

**Comment by Moondarker** [comment]
`Line 85: Cannot set property 'isOpen' of undefined`
Sugar is good tho

**Comment by u9g** [comment]
> `Line 85: Cannot set property 'isOpen' of undefined`
> 
> Sugar is good tho 

a. there is nothing on line 85
b. sugar?

File: lib/playerlist.js [hunk 1]
```diff
@@ -59,39 +59,30 @@ class PlayerList extends LitElement {
...
     function showList (shouldShow = true) {
-      if (shouldShow) {
-        playerList.style.display = 'block'
-      } else {
-        playerList.style.display = 'none'
-      }
-
-      self.isOpen = shouldShow
+      playerList.style.display = shouldShow ? 'block' : 'none'
+      this.isOpen = shouldShow
```
Comment (Moondarker) [inline lib/playerlist.js:85]: Cannot set property 'isOpen' of undefined (this is line 85)

**Comment by Moondarker** [comment]
And sugar = syntax sugar, yeah

**Comment by u9g** [comment]
I'm just confused about why self works but this doesnt

**Comment by Moondarker** [comment]
Because `this` changes depending on the context, and `const self = this` is just a way to save the context we want for later use

**Comment by u9g** [comment]
> Because `this` changes depending on the context, and `const self = this` is just a way to save the context we want for later use

ohh i see

**Comment by extremeheat** [comment]
Just use an arrow function and the `this` context will be passed down to the child function

File: lib/playerlist.js (hunk 1 above)
Comment (rom1504) [reply lib/playerlist.js:85]: this needs fixing

Comment (u9g) [reply lib/playerlist.js]: I think it's fixed now

**Review by rom1504 — APPROVED** [review]

**Comment by rom1504** [comment]
this is slightly better than before

however it can be improved a lot more, here is how if you're interested https://github.com/PrismarineJS/prismarine-web-client/pull/94/files#r593996913

**Pull request merged**